pkg/controller/clone-controller.go

```diff
@@ -240,7 +240,7 @@ func (cc *CloneController) syncPvc(key string) error {
 		return nil
 	}
 
-	if !checkPVC(pvc, AnnCloneRequest) || metav1.HasAnnotation(pvc.ObjectMeta, AnnCloneOf) {
+	if !checkPVC(pvc, AnnCloneRequest) || pvc.DeletionTimestamp != nil || metav1.HasAnnotation(pvc.ObjectMeta, AnnCloneOf) {
```

Comment:
How did this change to clone controller come up?

Reply:
Because I changed checkPVC to not include the check for the timestamp, I didn't want to alter the (potentially broken) behavior of the clone, so I had to add that back in this call.

Reply:
Note, I am looking at rewriting the clone controller as well, and in my initial investigation, I think that will get refactored away. Trying to figure out how to delete the source pod, once the upload server has deleted the target pod.